packages/story-editor/src/components/library/panes/pageTemplates/templateSave.js

```diff
@@ -37,7 +37,7 @@ import { useAPI } from '../../../../app/api';
 import { useStory } from '../../../../app/story';
 import { focusStyle } from '../../../panels/shared';
 import isDefaultPage from '../../../../utils/isDefaultPage';
-import { ReactComponent as Icon } from './images/illustration.svg';
+import { default as Icon } from './images/illustration.svg';
```

Comment:
So could we just do `import Icon from '...'` now everywhere?

Reply:
💭 I think we could!